syntax = "proto2";

message response_msg1
{
	required string info = 1;
}